lib/aca_entities/families/family_member.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Families
+    class FamilyMember < Dry::Struct
+      # TODO: Person hbx_id until we add hbx_id on family member
```

Comment:
EA currently assigns hbx_id to FamilyMember.  This isn't the Person hbx_id

Reply:
Removed comment, hbx_id field need to, add to enroll family member modal, currently hbx_id field on FM model